api/README.md

```diff
@@ -753,3 +754,74 @@ empty string, which will yield all test results for the selected runs.
 ```
 
 </details>
+
+## Metadata results
+
+### /api/metadata
+
+This endpoint accepts POST and GET requests.
+
+- GET request returns Metadata Link Information by product, and requires product parameters;
+
+- POST request searches Metadata Link by link url, and requires product parameters and payload.
+
+__URL Parameters__
+
+__`product`__ : browser[version[os[version]]]. e.g. `chrome-63.0-linux`
+
+#### JSON Payload
```

Comment:
This (`link`) is a query parameter, not part of the JSON payload?

Reply:
Nope, link is a part of payload. I think it could be either in the query parameter or a part of the JSON payload, but I did so because that is what searchcache does.

Reply:
searchcache has a complicated JSON blob; this example doesn't imply a JSON blob. Is `link` a field on a JSON object?